package.json

```diff
@@ -78,12 +78,12 @@
     "@justeattakeaway/pie-webc-testing": "0.13.0",
     "@justeattakeaway/pie-wrapper-react": "0.14.1",
     "@justeattakeaway/stylelint-config-pie": "0.7.0",
-    "@percy/cli": "1.26.3",
-    "@percy/playwright": "1.0.4",
-    "@playwright/experimental-ct-react": "1.32.3",
-    "@playwright/test": "1.32.3",
+    "@percy/cli": "1.28.8",
+    "@percy/playwright": "1.0.5",
```

Comment:
QQ (don't want to scope creep): Will we be updating to `1.0.6` soon?

Reply:
Updated and ran tests and looks all ok so will keep updated.